app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -219,21 +214,77 @@ var RepositoryDatatable = (function(global) {
 
       updateDataTableSelectAllCtrl();
 
-      e.stopPropagation();
+      ev.stopPropagation();
       updateButtons();
       // Update number of selected records info
       $('#selected_info').html(' (' + rowsSelected.length + ' entries selected)');
     });
 
     // Handle click on "Select all" control
-    $('.dataTables_scrollHead input[name="select_all"]').change(function(e) {
+    $('.dataTables_scrollHead input[name="select_all"]').change(function(ev) {
       if (this.checked) {
         $('.repository-row-selector:not(:checked)').trigger('click');
       } else {
         $('.repository-row-selector:checked').trigger('click');
       }
       // Prevent click event from propagating to parent
-      e.stopPropagation();
+      ev.stopPropagation();
+    });
+  }
+
+  function checkAvailableColumns() {
+    $.ajax({
+      url: $(TABLE_ID).data('available-columns'),
+      type: 'GET',
+      dataType: 'json',
+      success: function(data) {
+        var columnsIds = data.columns;
+        var presentColumns = $(TABLE_ID).data('repository-columns-ids');
+        if (!_.isEqual(columnsIds.sort(), presentColumns.sort())) {
+          alert($(TABLE_ID).data('columns-changed'));
```

Comment:
I think you forgot to remove this alert

Reply:
Why? It is a part of column checking logic.